grails-app/controllers/org/pih/warehouse/order/OrderController.groovy

```diff
@@ -1096,6 +1096,66 @@ class OrderController {
     }
 
     def orderSummary = {
+        Order order = Order.get(params.id)
+        def orderItems = order?.orderItems
+        render(template: "orderSummary", model: [
+            isPurchaseOrder: order.isPurchaseOrder,
```

Comment:
@jmiranda I started moving things to the controller that are needed by the template and since there are used properties from order and order items I decided to leave it as is -> fetch order and pull order items (and other required data) from it instead of fetching only order items.